snooty.toml

```diff
@@ -25,6 +25,7 @@ sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 driver-short = "PyMongo"
 driver-long = "PyMongo, the MongoDB synchronous Python driver,"
 language = "Python"
+mdb-server = "MongoDB Server"
```

Comment:
Q: since you are adding this here, are there other places that you could use this constant?

Reply:
good call